engine/lib/plugins.php

```diff
@@ -522,8 +522,6 @@ function _elgg_plugins_init() {
 	elgg_register_action('admin/plugins/deactivate_all', '', 'admin');
 
 	elgg_register_action('admin/plugins/set_priority', '', 'admin');
-
-	elgg_register_library('elgg:markdown', elgg_get_root_path() . 'vendors/markdown/markdown.php');
```

Comment:
Consider a 3rd-party plugin that calls `elgg_load_library('elgg:markdown')`. if you leave the lib as is, PHP will fatal as the old lib tries to redefine markdown(). If you remove the lib, Elgg will exception out.